build-system/global-configs/prod-config.json

```diff
@@ -13,7 +13,6 @@
   "amp-action-macro": 1,
   "amp-ad-ff-adx-ady": 0.01,
   "amp-auto-ads-adsense-holdout": 0.1,
-  "amp-auto-ads-no-op-experiment": 0.05,
```

Comment:
Is it still the case that we should stagger this? At some point in time, our config releases were instantaneous, whereas our binary releases were slightly staggered. Therefore we made a point of making sure that the dependent code reached production prior to removing the experiment from prod-config.json / canary-config.json. I'm not sure if that's still the case, but it would be worth verifying. =)